Upstream: https://svn.boost.org/trac/boost/ticket/4994
Debian-Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=622021
Description: Python 3.2 compatibility, based on patch by Matthew Bradbury.

--- boost1.46-1.46.1.orig/libs/python/src/converter/builtin_converters.cpp
+++ boost1.46-1.46.1/libs/python/src/converter/builtin_converters.cpp
@@ -432,5 +432,8 @@
           {
               int err = PyUnicode_AsWideChar(
-                  (PyUnicodeObject *)intermediate
+#if PY_VERSION_HEX < 0x03020000
+                  (PyUnicodeObject *)
+#endif
+                    intermediate
                 , &result[0]
                 , result.size());
